test/library/standard/Random/RandomStreamInterface/choiceTest.chpl

```diff
@@ -84,7 +84,7 @@ record R {
 }
 
 proc testArray(stream, arr: [] ?eltType, size:?sizeType=none, replace=true, prob:?probType=none, trials=10000) throws {
-  var counts = new map(eltType, int);
+  var counts = new map(int, int);
```

Comment:
I think this is reverting a change made on master.

Reply:
This seems to be necessary, as now we have to maintain a key -> value pair of (int, int) instead of (element type, int).
(Considering the fact that we have to check for `indices` rather than `elements` now.)

Reply:
huh, strange. Can you pull or rebase with master to confirm the latest changes to `Random.choice()` work with this change?

Reply:
I rebased with master(after pulling from the upstream master), and all tests in ` test/library/standard/Random/RandomStreamInterface/choiceTest.chpl` pass. 

I am sorry, but I am getting a bit confused, why is this change so surprising? Currently, in `choice` we are returning elements of the array, which can be `int` or `char`, so we used `eltType` of the array. But in this version of `choice` we are returning indices, that should be `int` so we used `int` type for the map key, which otherwise will throw an error.